scripts/feed/PostInteraction.js

```diff
@@ -3,8 +3,32 @@ handles the rendering and interaction of a post element
 */
 
 // import deletePost, favoritePost from ./data/TransientState.js
+import { deletePost, favoritePost } from '../data/TransientState.js';
+
 // attach 2 click event listeners -- both outside any function:
     // Delete button
     // Favorite button
-// declare PostInteraction function
-// export PostInteraction function to PostList
+document.addEventListener('click', handleDeleteButtonClick);
```

Comment:
Because the 'handleDeleteButtonClick' callback function is defined inside of the 'PostInteraction' function, I don't think this event listener will be able to access it. @chesney-hardin maybe you could confirm/deny that?